aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLRequestFactory.java

```diff
@@ -468,4 +458,17 @@ private static String getModelFields(Class<? extends Model> clazz, int levelsDee
 
         return result.toString();
     }
+
+    static final class InvalidGraphQLRequestRuntimeException extends RuntimeException {
+
+        private static final long serialVersionUID = 1L;
+
+        InvalidGraphQLRequestRuntimeException(final String message, final Exception rootCause) {
+            super(message + "\n" + AmplifyException.REPORT_BUG_TO_AWS_SUGGESTION, rootCause);
```

Comment:
Don't use Report bug for these types

Reply:
Yea, the `AmplifyException` has the strongly modeled recovery exception. I don't think we should inline the recovery suggestions into causes. I'm imagining that this `InvalidGraphQLRequestRuntimeException` gets composed into an `ApiException`, before being returned to the customer. If that's so, then we'd have _two_ recovery suggestions. This one, then the one in `ApiException`.